integration/benchmark/output.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Would it be in the repos?

Reply:
This is intended as an example, and as input you might pass to the comparison (or hypothetical regression check) scripts. At least two such examples will be needed if we add unit tests for those scripts. I should probably rename it, at least. Perhaps the example in the README is sufficient, and further examples would exist within the data for unit tests, so there isn't an extra file floating around?